presto-docs/src/main/sphinx/release/release-0.214.rst

```diff
@@ -0,0 +1,64 @@
+=============
+Release 0.214
+=============
+
+General Changes
+---------------
+
+* Fix history leak in coordinator for failed or canceled queries.
+* Fix memory leak related to query tracking in coordinator that was introduced
+  in 0.213.
+* Fix planning failures when lambdas are used in join filter expression.
+* Send proper error response to client for certain types of errors during query
+  creation.
+* Improve error message when comparator provided to :func:`array_sort` function
+  does not satisfy the required contract.
+* Improve performance for lookup operations on map data types.
+* Improve planning and query times for queries with ``TINYINT``, ``SMALLINT``
+  and ``VARBINARY`` literals.
+* Maintain order of predicates and join tables in the query plan.
+* Make distributed ``ORDER BY`` and aggregation unspill progress on early yield
```

Comment:
Don't understand this. Which commit is it?

Reply:
Seems to be introduced by c528f8